src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -176,6 +176,19 @@ The command that should be run by the nfs-fixer init container to correct the pe
 {{- end }}
 {{- end }}
 
+{{ define "confluence.volumeMounts" }}
+- name: local-home
+  mountPath: {{ .Values.volumes.localHome.mountPath | quote }}
+- name: local-home
+  mountPath: {{ .Values.confluence.accessLog.mountPath | quote }}
+  subPath: {{ .Values.confluence.accessLog.localHomeSubPath | quote }}
+- name: shared-home
+  mountPath: {{ .Values.volumes.sharedHome.mountPath | quote }}
+  {{- if .Values.volumes.sharedHome.subPath }}
+  subPath: {{ .Values.volumes.sharedHome.subPath | quote }}
+  {{- end }}
+{{ end }}
```

Comment:
The `volumeMounts` section of `statefulset.yaml` was getting a bit verbose, so I've extracted it into its own template fragment. It's the same as before, but now with the 2nd `local-home` mount added in.